src/station_cmd.cpp

```diff
@@ -1392,8 +1528,29 @@ CommandCost CmdBuildRailStation(DoCommandFlags flags, TileIndex tile_org, RailTy
 		if (ret.Failed()) return ret;
 	}
 
-	/* Check if we can allocate a custom stationspec to this station */
 	const StationSpec *statspec = StationClass::Get(spec_class)->GetSpec(spec_index);
+	TileIndexDiff tile_delta = TileOffsByAxis(axis); // offset to go to the next platform tile
+	TileIndexDiff track_delta = TileOffsByAxis(OtherAxis(axis)); // offset to go to the next track
+
+	std::vector<uint8_t> layouts(numtracks * plat_len);
+	GetStationLayout(layouts.data(), numtracks, plat_len, statspec);
+
+	{
```

Comment:
Doing something fancy with scope, or orphaned braces from a rebase gone wrong?

If the former, could use a comment explaining what this section does.